openbts/components.py

```diff
@@ -41,6 +41,40 @@ def monitor(self):
     }
     return self._send_and_receive(message)
 
+  def tmsis(self):
+    """Gets all active subscribers from the TMSI table
+
+    Will return a list of objects of the form: {
+            'A5_SUPPORT' : '',
```

Comment:
more nits: can you explain some / all of these keys in comments?  Or link to docs?